packages/framework/src/Resources/views/Admin/Content/Order/preview.html.twig

```diff
@@ -1,36 +1,33 @@
-<div class="text-left">
-    <h3 class="window__title">{{ 'Order Nr. %orderNumber%'|trans({'%orderNumber%' : order.number}) }}</h3>
-
-    <table class="table-main">
-        <thead>
-            <tr>
-                <th class="table-col-60">{{ 'Item name'|trans }}</th>
-                <th class="table-col-20 text-center">{{ 'Amount'|trans }}</th>
-                <th class="table-col-20 text-right">{{ 'Price including VAT'|trans }}</th>
-            </tr>
-        </thead>
-        <tbody>
-            {% for item in order.items %}
-                <tr>
-                    <td>
-                        {% if item.typeTransport %}
-                            {{ 'Shipping'|trans }} -
-                        {% elseif item.typePayment %}
-                            {{ 'Payment'|trans }} -
-                        {% endif %}
-                        {{ item.name }}
-                    </td>
-                    <td class="text-center">{{ item.quantity }} {{ item.unitName }}</td>
-                    <td class="text-right">{{ item.totalPriceWithVat|priceWithCurrency(order.currency) }}</td>
-                </tr>
-            {% endfor %}
-        </tbody>
-        <tfoot>
-            <tr>
-                <td><strong>{{ 'Total including VAT'|trans }}</strong></td>
-                <td></td>
-                <td class="text-right"><strong>{{ order.totalPriceWithVat|priceWithCurrency(order.currency) }}</strong></td>
-            </tr>
-        </tfoot>
-    </table>
+<div class="window-content__heading text-left pl30">
```

Comment:
This element is not rendered in `.window-content`, so according to BEM it must not have `window-content__heading` class.

Also, the heading doesn't look quite well imo, maybe there's something broken...
![image](https://user-images.githubusercontent.com/10008612/61797578-f2aea480-ae27-11e9-833c-781f6c0bf036.png)

When I've tried to add a wrapping `.window-content` the heading was rendered over the table header because of the `position: absolute;`.

Reply:
I tried to unify display in hover and display in popup. So i used `window-content__heading` for both.

Reply:
I'm asking because of the big red warning about not doing this in our own CSS documentation:
![image](https://user-images.githubusercontent.com/10008612/61849191-c3904580-aeb0-11e9-9fe5-f5da19e2b51c.png)

Could you please explain why is it OK in this case?

Reply:
After a discussion it seems that it's no OK, waiting for reimplementation.